docs/dev/building.md

```diff
@@ -22,37 +22,45 @@ git clone https://github.com/scylladb/scylla
 cd ./scylla
 git submodule update --init --force --recursive
 ./tools/toolchain/dbuild ./configure.py --mode=<mode>
-./tools/toolchain/dbuild ninja
+./tools/toolchain/dbuild cmake --build build
 ```
 
 where `mode` is `dev` for development builds, `release` for release builds, and `debug` for debug builds.
 
-Alternatively, you can also pass the `mode` directly to the `ninja` command:
+Alternatively, you can also pass the `config` directly to the `cmake` command:
 
 ```console
 git submodule update --init --force --recursive
 ./tools/toolchain/dbuild ./configure.py
-./tools/toolchain/dbuild ninja <mode>
+./tools/toolchain/dbuild cmake --build build --config <config>
```

Comment:
Ditto: possibly right (I couldn't say), but not consistent with the recommendation in README.md (and with my own use thus far). Please confirm.

Reply:
i see what you mean now -- "recommendation". i intentionally avoid talking too much about CMake in "README.md" to avoid the potential contentious CMake versus ninja discussions. instead, i prefer keeping the `README.md` neutral, and leave the discussion for future. 

while I'm still learning the nuances of these technical discussions, I believe this PR is important for the CMake migration path. i suggest we merge it now to maintain momentum, and we can refine the documentation's language and scope later as we gather more community feedback.